### PR TITLE
FacilityTransaction queue: bugfixes

### PR DESCRIPTION
**Pull request by Theodlz**
@mcoughlin feel free to add to this based on what we discussed on slack.

### CODE REVIEW

**Comment by mcoughlin** [comment]
@Theodlz Seems to be ingesting alright after your suggested changes. Please lemme know.

File: services/facility_queue/facility_queue.py [hunk 1]
```diff
@@ -237,29 +257,39 @@ def service(queue):
...
-                            except Exception:
-                                status = 'In progress: Light curve not yet available. Waiting for it to complete.'
-                                followup_request.status = status
-                                log(f'Job {req.id}: {status}')
-                                req.last_query = datetime.utcnow()
+                            exitcode = row['exitcode']
+                            exitcode_text = ZTF_PHOTOMETRY_CODES[exitcode]
+
+                            if exitcode in [63, 64, 65, 255]:
+                                req.status = f'No photometry available: {exitcode_text}'
```
Comment (Theodlz) [inline services/facility_queue/facility_queue.py]: @mcoughlin is that what we want, or do we want that be the followup request status is set to 'no photometry available', while the facility transaction is set to complete? That way its not fetched anymore when we restart the queue.

File: skyportal/facility_apis/atlas.py [hunk 2]
```diff
@@ -312,7 +319,19 @@ def delete(request, session):
...
+
+        transaction = (
+            session.query(FacilityTransactionRequest)
+            .filter(FacilityTransactionRequest.followup_request_id == request.id)
+            .first()
+        )
+        if transaction is not None:
+            if transaction.status == "complete":
+                raise ValueError('Request already complete. Cannot delete.')
+            session.delete(transaction)
```
Comment (Theodlz) [inline skyportal/facility_apis/atlas.py:332]: Nice. Thanks!

File: services/facility_queue/facility_queue.py (hunk 1 above)
Comment (mcoughlin) [reply services/facility_queue/facility_queue.py]: yeah I had fixed that in the last commit.

**Pull request merged**